src/tv2-common/helpers/graphics/viz/index.ts

```diff
@@ -59,6 +59,26 @@ export function GetInternalGraphicContentVIZ(
 	}
 }
 
+export function findShowName(
```

Comment:
It appears that this method is only used in here. If that is true, then I think we shouldn't export

Reply:
I removed `export`